### PR TITLE
doc: fix image references

### PR DESCRIPTION
**Pull request by dbkinder**
* change image file names not to include spaces
* use markdown syntax for image references instead of embedded HTML

### CODE REVIEW

File: authN-authZ/auth-istio/README.md [hunk 1]
```diff
@@ -221,31 +221,31 @@ The user management is done via Keycloak and the configuration steps look like t
 
 1. Create a new realm named `chatqna` within Keycloak.
 
-<img src="./docs/create_realm.png" width="600" height="300">
+   ![create realm](./docs/create_realm.png)
 
-2. Create a new client called `chatqna` and set `Client authentication` to 'On'. Set "http://chatqna-ui.com:${INGRESS_PORT}/*" in the `Valid redirect URIs` part. Note that `INGRESS_PORT` and `INGRESS_HOST` shall be exported following the guide [here](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports). Under the Credentials tab you will now be able to locate `<your client's secret>`, which will be used in the oauth2-proxy configs.
+2. Create a new client called `chatqna` and set `Client authentication` to 'On'. Set "http://chatqna-ui.com:${INGRESS_PORT}/\*" in the `Valid redirect URIs` part. Note that `INGRESS_PORT` and `INGRESS_HOST` shall be exported following the guide [here](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports). Under the Credentials tab you will now be able to locate `<your client's secret>`, which will be used in the oauth2-proxy configs.
```
Comment (Ruoyu-y) [inline authN-authZ/auth-istio/README.md:226]: User need to set `http://chatqna-ui.com:${INGRESS_PORT}/*` in the field. Suggest to remove the change.

**Review by Ruoyu-y — CHANGES_REQUESTED** [review]

File: authN-authZ/auth-istio/README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ The istio ingress gateway will be used to access the chatQnA service in differen
 
 Authentication and authorization are essential for securing microservices architectures. Using Bearer JWT tokens for these processes ensures that only authenticated users with valid tokens can access specific services, protecting sensitive data. Authentication verifies user identity, while authorization controls their permissions. This layered approach not only prevents unauthorized access but also provides detailed control over service interactions, maintaining system security and compliance. Here we leverage Istio mechanisms together with Bearer JWT tokens to fulfill that.
 
-<img src="./docs/OPEA auth flow with OIDC provider.png" width="700" height="300">
+![OPEA auth flow with OIDC provider](./docs/OPEA_auth_flow_with_OIDC_provider.png)
```
Comment (xiguiw) [inline authN-authZ/auth-istio/README.md:48]: Is the image size and layout changed if remove image width and height setting?
The width and height could be set with HTML.

File: authN-authZ/auth-istio/README.md (hunk 1 above)
Comment (dbkinder) [reply authN-authZ/auth-istio/README.md]: Line is restored (change removed)

File: authN-authZ/auth-istio/README.md (hunk 2 above)
Comment (dbkinder) [reply authN-authZ/auth-istio/README.md:48]: Yes, the image size changes but still looks OK without manually setting the width and height.

**Review by Ruoyu-y — APPROVED** [review]
LGTM

**Review by lianhao — APPROVED** [review]

**Pull request merged**